libraries/disp3D/engine/view/orbitalcameracontroller.h

```diff
@@ -136,6 +151,8 @@ class DISP3DSHARED_EXPORT OrbitalCameraController : public Qt3DExtras::QAbstract
 
     float m_rotationInversFactor = 1.0f;             /**< The factor used to invers the camera rotation. */
```

Comment:
Although you did not add this line, please rename `m_rotationInversFactor` to `m_fRotationInverseFactor`. the `f` hereby indicates a float value